modules/siddhi-core/src/main/java/org/wso2/siddhi/core/query/input/stream/StreamRuntime.java

```diff
@@ -30,7 +30,7 @@
 
     List<SingleStreamRuntime> getSingleStreamRuntimes();
 
-    StreamRuntime clone(String key);
+    StreamRuntime clone(String queryName, String key);
```

Comment:
@miyurud This is an API change. Is there any place that we discuss and approved this change. I need to know We discussed the implications before approving. I would rather keep the old method and introduce a new overloaded one. @suhothayan please override this review if you already discussed this.